src/components/WebCLI/WebCLI.tsx

```diff
@@ -173,7 +173,7 @@ const WebCLI = ({
       const origin = connection.current?.address
         ? new URL(connection.current?.address)?.origin
         : null;
-      const macaroons = origin ? bakery.storage.get(origin) : null;
+      const macaroons = origin ? bakery.storage.get(`${origin}/api`) : null;
```

Comment:
This needed to match the format as is provided in config.js which ends with `/api`